src/kernel/arch/x86/paging.zig

```diff
@@ -361,9 +363,27 @@ pub fn unmap(virtual_start: usize, virtual_end: usize, dir: *Directory) (std.mem
 /// Called when a page fault occurs.
 ///
 /// Arguments:
-///     IN state: *arch.InterruptContext - The CPU's state when the fault occurred.
-///
-fn pageFault(state: *arch.InterruptContext) void {
+///     IN state: *arch.CpuState - The CPU's state when the fault occurred.
+///
+fn pageFault(state: *arch.CpuState) u32 {
```

Comment:
Effort lel, So move the change to returning a stack pointer to a diff PR

Reply:
Oh I meant the changes to the page fault function rather than the return type change! The return type of course needs to change in this PR. I just think changing what the pageFault function actually does would be nice to have in a different PR.

Reply:
Ah, ok, can do :)